.gitignore

```diff
@@ -1,5 +1,5 @@
-# Created by https://www.toptal.com/developers/gitignore/api/yarn,node,visualstudiocode,macos,linux,windows,astro
-# Edit at https://www.toptal.com/developers/gitignore?templates=yarn,node,visualstudiocode,macos,linux,windows,astro
+# Created by https://www.toptal.com/developers/gitignore/api/pnpm,node,visualstudiocode,macos,linux,windows,astro
+# Edit at https://www.toptal.com/developers/gitignore?templates=pnpm,node,visualstudiocode,macos,linux,windows,astro
```

Comment:
This doesn't seem accurate anymore since it looks like a text replacement rather then a generated file